suite-native/message-system/src/selectors.ts

```diff
@@ -0,0 +1,30 @@
+import { memoize } from 'proxy-memoize';
+
+import { MessageSystemRootState } from '@suite-common/message-system';
+
+export const selectAllActiveMessageIds = memoize((state: MessageSystemRootState) => {
+    const { validMessages, dismissedMessages } = state.messageSystem;
+
+    // At the moment, in suite-native/message-system we are not distinguishing between different message
+    // categories. For that reason all categories are merged to a one single array.
+    const validMessageIds = Object.values(validMessages).reduce(
+        (acc, messages) => acc.concat(messages),
+        [],
+    );
+
+    const nonDismissedValidMessageIds = validMessageIds.filter(id => !(id in dismissedMessages));
+
+    return nonDismissedValidMessageIds;
```

Comment:
maybe you forgot to use `comparePriority` for sorting here? I see you added the export but then you don't use it at all?

Reply:
You were right, i forgot to use it 🫢. But since the fixup https://github.com/trezor/trezor-suite/commit/1fc0085db992732ed544631274226788618721b2 this selector does not exist anymore, so this comment is not relevant anymore.